internal/params/params_test.go

```diff
@@ -186,15 +310,13 @@ func Test_parser_Parse(t *testing.T) {
 			if err := test.checkFunc(test.want, got, got1, err); err != nil {
 				tt.Errorf("error = %v", err)
 			}
-
 		})
 	}
 }
 
 func Test_data_ConfigFilePath(t *testing.T) {
 	type fields struct {
 		configFilePath string
-		showVersion    bool
```

Comment:
I think we don't need to remove this line.

Reply:
could you please update GitHub paga?
In my environment, there is no this line.

Reply:
Is it generated by the `gotests` command?